packages/crypto/package.json

```diff
@@ -44,13 +44,13 @@
     "@noble/ciphers": "0.5.3",
     "@noble/curves": "1.4.0",
     "@noble/hashes": "1.4.0",
-    "@turnkey/encoding": "workspace:*",
     "bs58check": "3.0.1",
     "bs58": "^5.0.0"
   },
   "devDependencies": {
     "jest": "29.7.0",
     "@turnkey/encoding": "workspace:*",
-    "@turnkey/http": "workspace:*"
```

Comment:
interesting that the removal of `@turnkey/sdk-server` isn't in this diff... for some reason? maybe github UI thing?